modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1583 @@
+
+ifndef::env-kubernetes[]
+[[kerberos]]
+==== GSSAPI (Kerberos)
+
+NOTE: Kerberos authentication requires an xref:get-started:licenses.adoc[Enterprise license]. To upgrade, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^].
+
+To configure Kerberos authentication, use a keytab, which contains credentials for the service.
+
+. <<enable-authentication, Enable SASL authentication>> if it's not already enabled.
+
+. Enable <<SASL/SCRAM>> for the Kafka API.
++
+For Kerberos authentication, Redpanda requires that SASL/SCRAM be enabled for connectivity by `rpk`, Redpanda Console, and other Redpanda products. Operating with Kerberos only is not a supported configuration.
+
+. If the keytab is not in the default location, then set its location:
++
+[,bash]
+----
+rpk cluster config set sasl_kerberos_keytab <path>
+----
+
+. If the krb5.conf file is not in the default location, then set its location:
++
+[,bash]
+----
+rpk cluster config set sasl_kerberos_config <path>
+----
+
+. Define the primary of the Kerberos SPN to be used by Redpanda with the given keytab. Default is `redpanda`.
++
+[,bash]
+----
+rpk cluster config set sasl_kerberos_principal <name>
+----
+
+. Set `sasl_kerberos_principal_mapping`. This maps Kerberos user principal names (UPNs) onto Redpanda principals used in the ACLs. For example:
++
+[,bash]
+----
+rpk cluster config set sasl_kerberos_principal_mapping '["RULE:[1:$1@$0](.*@MYDOMAIN.COM)s/@.*//","DEFAULT"]'
+----
++
+By default, Redpanda matches the primary of the Kerberos UPN of the user. Each rule has the following format:
++
+* `RULE:[n:string](regexp)s/pattern/replacement/g/c` where:
+** `n` is an integer that indicates how many components the target principal should have.
+** If this matches, then a string is formed from `string`, substituting the realm of the principal for `$0` and the `n``'th component of the principal for `$n`. (For example, if the principal is `johndoe/admin@realm.com`, then `[2:$2$1foo]` results in the string `adminjohndoefoo`.)
+** If this string matches `regexp`, then the `s//[g]` substitution command is run over the string.
+** `g` is optional. It causes the substitution to be global over the string, instead of replacing only the first match in the string.
+** `c` is optional. It can be either `/L` or `/U` to make the match lowercase or uppercase.
+* `DEFAULT` The principal name is used as the local user name. If the principal has more than one component or is not in the default realm, then the conversion fails.
++
+Examples of a Kerberos UPN without a host (`jdoe@EXAMPLE.COM`) and with a host (`jdoe/host@EXAMPLE.COM`):
++
+|===
+| Translation | `jdoe@EXAMPLE.COM` | `jdoe/host@EXAMPLE.COM`
+| `[1:$1@$0]`
+| `jdoe@EXAMPLE.COM`
+| Rule does not matchfootnote:two-components[Rule does not match because there are two components in principal name `jdoe/host@EXAMPLE.COM`.]
```

Comment:
So the actual notes for 1 and 2 here appear at the very end of the topic. If you encounter them there, it is a bit puzzling for sure, and just not a good user experience with this content. Is there any way you can position right underneath this table?

Reply:
We could do something custom. But in this case, I think it's simpler to not have footnotes and just include the reasons in the table cells.